modules/apigee_edge_teams/apigee_edge_teams.links.action.yml

```diff
@@ -1,18 +1,18 @@
 apigee_edge_teams.team.add_members:
   route_name: entity.team.add_members
-  title: 'Add members'
+  title: 'Add team members'
   appears_on:
     - 'entity.team.members'
 
 apigee_edge_teams.team_app.add_form_for_team:
   route_name: entity.team_app.add_form_for_team
-  title: 'Add app'
+  title: 'Add team app'
```

Comment:
See 1st comment in apigee_edge_teams.links.action.yml.

Reply:
Updated PR.